packages/entities/entities-shared/src/components/entity-toggle-modal/EntityToggleModal.cy.ts

```diff
@@ -63,26 +63,26 @@ describe('<EntityToggleModal />', () => {
       })
 
       it('should disable action button when |onConfirm| is pending', () => {
-        cy.get('.k-prompt-proceed').should('not.be.disabled')
+        cy.get('button[data-testid="modal-action-button"]').should('not.be.disabled')
         // eslint-disable-next-line cypress/unsafe-to-chain-command
-        cy.get('.k-prompt-proceed').click().then(() => {
-          cy.get('.k-prompt-proceed').should('be.disabled')
+        cy.get('button[data-testid="modal-action-button"]').click().then(() => {
+          cy.get('button[data-testid="modal-action-button"]').should('be.disabled')
           // eslint-disable-next-line cypress/no-unnecessary-waiting
           cy.wait(kConfirmDelay + 10).then(() => {
-            cy.get('.k-prompt-proceed').should('not.be.disabled')
+            cy.get('button[data-testid="modal-action-button"]').should('not.be.disabled')
           })
         })
       })
 
-      it('should not |onConfirm| more than once when |onConfirm| is pending', () => {
+      it.skip('should not |onConfirm| more than once when |onConfirm| is pending', () => {
         cy.wrap(onConfirmSpy).should('not.be.called')
         // eslint-disable-next-line cypress/unsafe-to-chain-command
-        cy.get('.k-prompt-proceed').click().then(() => {
+        cy.get('button[data-testid="modal-action-button"]').click().then(() => {
           cy.wrap(onConfirmSpy).should('be.calledOnce')
           // eslint-disable-next-line cypress/no-unnecessary-waiting
           cy.wait(10).then(() => {
             // eslint-disable-next-line cypress/unsafe-to-chain-command
-            cy.get('.k-prompt-proceed').click({ force: true }).then(() => {
+            cy.get('button[data-testid="modal-action-button"]').click({ force: true }).then(() => {
```

Comment:
can you replace these button selectors with `cy.getTestId` ?

Reply:
Updated